recipes/openfast_io/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "openfast_io" %}
+{% set version = "4.0.2" %}
+{% set python_min = "3.11" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/openfast_io-{{ version }}.tar.gz
+  sha256: 566d00e930216a92f74e4b991da687cb9c93028fa3783279db04e0be066c35b5
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - hatchling
+    - hatch-vcs
+    - pip
+  run:
+    - python >={{ python_min }}
+    - numpy >1.0
+    - pandas >2.0
+    - ruamel.yaml >0.18
+    - deepdiff >8.0
+
+test:
+  imports:
+    - openfast_io
+  commands:
+    - pip check
+  requires:
+    - python {{ python_min }}
+    - pip
+
+about:
+  home: https://github.com/openfast/openfast
+  license: Apache-2.0
+  license_file: LICENSE
```

Comment:
The license file is missing from the source distribution. You should add a manual copy here and raise an issue upstream to fix it so the manual copy can be removed in a future release:
https://conda-forge.org/docs/maintainer/adding_pkgs/#packaging-the-license-manually